sale_automatic_workflow_stock_location_route/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This allow to automatically set a specific route on sale lines with product delivered by stock move, when the sale is confirmed.
```

Comment:
I would rephrase, like on sale lines with physical (or any better way) product. I think "delivered by stock move" is not very clear for a none odoo developer person. I may be wrong.